cdk/cdk/consoleme_ecs_service/nested_stacks/config_stack.py

```diff
@@ -12,7 +12,7 @@
 from aws_cdk import aws_logs as logs
 from aws_cdk import core as cdk
 from aws_cdk import custom_resources as cr
-from aws_cdk.aws_lambda_python import PythonFunction as lambda_python
+from helpers import create_dependencies_layer
```

Comment:
Is it possible to make this an absolute import? Isort thinks `helpers` is a third-party library

Reply:
I'm trying to avoid using another tool for dependencies resolving. Maybe ill try to generate the requirements file using `pipfile lock`

Reply:
I'm kinda struggling to do it, any suggestion?

I noticed that also `constants` is a third party according to `isort` so i'm wondering what can we do. Not too familiar with `isort` anyways.

Reply:
I have some idea, I'll do it tonight